pkg/trace/api/api.go

```diff
@@ -401,24 +402,64 @@ func (r *HTTPReceiver) handleStats(w http.ResponseWriter, req *http.Request) {
 	var buf strings.Builder
 	for _, group := range in.Stats {
 		for _, b := range group.Stats {
-			newb := stats.Bucket{
-				Start:    group.Start,
-				Duration: group.Duration,
-				Counts:   make(map[string]stats.Count),
+			b.Name, _ = traceutil.NormalizeName(b.Name)
+			b.Env = traceutil.NormalizeTag(b.Env)
+			b.Service, _ = traceutil.NormalizeService(b.Service, req.Header.Get(headerLang))
```

Comment:
note to not forget: when b.HTTPStatus is added, there is special normalization for it,
allowed values int in [100,600[